### ISSUE TITLE
nteract desktop does not find python kernel (windows 10)

### ISSUE THREAD

**Issue by pybokeh** [nteract/archived-desktop-app#4343]
**Application or Package Used**
nteract desktop

**Describe the bug**
nteract desktop does not connect or find my python kernel

**To Reproduce**
Step 1) python3 -m venv my_environment_name      # create a virtual environment
Step 2) my_environment_name/Scripts/activate.bat  # activate the virtual environment
Step 3) python -m pip install ipykernel          # install the python kernel (ipykernel) into the virtual environment
Step 4) python -m ipykernel install **--sys-prefix**
**NOTE:** without the --sys-prefix, I would get a: [WinError 5] Access is denied: 'C:\\ProgramData\\jupyter'   error message.
With the **--sys-prefix** added, I got the following message:
Installed kernelspec python3 in D:\Python37-32\envs\notebook\share\jupyter\kernels\python3
Step 5) nteract.exe --kernel python3
**Output of console or terminal:**
←[36m08:00:42.614←[0m ΓÇ║ args { _: [], kernel: 'python3', k: 'python3', '$0': 'nteract.exe' }←[0m
←[36m08:00:42.718←[0m ΓÇ║ Checking for update←[0m
←[36m08:00:42.809←[0m ΓÇ║ launching an empty notebook by default←[0m
←[36m08:00:46.097←[0m ΓÇ║ Update for version 0.13.0 is not available (latest version: 0.13.0, downgrade is disallowed).←[0m
←[0m08:00:46.099←[0m ΓÇ║ checkForUpdatesAndNotify called, downloadPromise is null←[0m

I also tried this at the console with my environment activated:
nteract.exe --kernel D:\Python37-32\envs\notebook\share\jupyter\kernels\python3

But nteract desktop still did not find or recognize my python kernel.

**Expected behavior**
nteract desktop would connect to python kernel

- OS: Windows 10
- Chrome
- 72.0.3626.121

**Comment by pybokeh** [comment]
OK, I think I found the problem.  Looks like nteract desktop isn't "smart" enough to look at other possible locations of the python kernel.

I issued this command at the command prompt: jupyter --paths
Which returns:
config:
    D:\.jupyter
    d:\python37-32\envs\notebook\etc\jupyter
    C:\ProgramData\jupyter
data:
    D:\.jupyter
    C:\Users\some_random_guy_on_Internet\AppData\Roaming\jupyter
    d:\python37-32\envs\notebook\share\jupyter
    C:\ProgramData\jupyter
runtime:
    C:\Users\some_random_guy_on_Internet\AppData\Roaming\jupyter\runtime

So what I did was copy my python kernel folder located at ```D:\Python37-32\envs\notebook\share\jupyter\kernels\python3``` to the ```D:\.jupyter``` folder and now nteract desktop finds my python kernel.

So it appears, nteract just needs to search other possible valid paths and also honor that there are people that install dependencies via the --sys-prefix, so it needs to look at the ```my_environment/share``` folder also.

**Comment by BenRussert** [comment]
> Looks like nteract desktop isn't "smart" enough

Can you check the GitHub issue search feature to see if it's "smart" enough to help you find our open kernel detection on Windows issue?

I didn't read the rest of your comments to know if it will help us, but maybe you can figure out how to discuss a problem constructively there.

**Comment by pybokeh** [comment]
@BenRussert I apologize, I did not know there was an already open issue https://github.com/nteract/nteract/issues/3670  Also, I think you mistook my comment as a jab towards nteract and that is not the case at all.  I would like to personally apologize if you took it that way.  I just meant that it looks like it just needed to search in other paths or directories.  Poor choice of words on my part.

I did see this [one](https://github.com/nteract/nteract/issues/3653) issue, but was closed as unresolved.

**Comment by BenRussert** [comment]
Cool @pybokeh, I get a little sensitive sometimes because many of us do all this for fun in our free time.


Do you think this is the same issue as #3670? What version of nteract are you on?

**Comment by pybokeh** [comment]
I was using nteract at work for the first time today and I won't be returning work until a week later. So I won't be able to get back to you until then.  But, I know I downloaded the .zip version since I don't have admin rights to my machine.  So I'm guessing it is version 0.13.  Would be curious what the output looks like for others when issuing the ```jupyter --paths``` command in that issue post and then have them check where their python kernel folder is actually installed.  That's how I figured out my problem.
